docs/batch.md

```diff
@@ -0,0 +1,86 @@
+# PREVIEW - Batch Messages (A.K.A. Fast Sync)
+
+```{warning}
+The `BATCH` message functionality is currently in preview and is subject to change.
+```
```

Comment:
```suggestion
```{warning}
The `BATCH` message functionality is currently in preview and is subject to change.
You can open a [GitHub issue](https://github.com/meltano/sdk/issues) or [join the discussion](https://github.com/meltano/sdk/discussions/963) on GitHub to provide feedback during the preview period.
```
```